### PR TITLE
ITSC V3 Release

### PR DESCRIPTION
**Pull request by EternalSH**
Hi.

This is the  V3 of the AWS Connector Quick Start. The key change is support for Kepware and Wonderware historian systems.
This version requires version V3 of the connection-agent, which we will provide as a ZIP archive. The Docker images are available on the DockerHub, from public registry.

### CODE REVIEW

File: templates/kepware-industrial-data-connector-master.yaml [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+      Use this parameter to specify the S3 bucket name you've created for your copy
+      of Quick Start assets, if you decide to customize or extend the Quick Start
+      for your own use. The bucket name can include numbers, lowercase letters, uppercase
+      letters, and hyphens, but should not start or end with a hyphen.
+    Type: String
+  QSS3KeyPrefix:
+    AllowedPattern: ^[0-9a-zA-Z-][0-9a-zA-Z-/]*/$
+    ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
+      uppercase letters, hyphens (-), and forward slash (/).
+    Default: quickstart-osisoft-pisystem2aws-connector/
```
Comment (sshvans) [inline templates/kepware-industrial-data-connector-master.yaml]: The default value for this parameter needs to be the name of the github repo 'quickstart-47lining-industrial-data-connector/'. Please updated it in all master and workload templates.

**Comment by EternalSH** [comment]
> @EternalSH Do you think you will be able to have the recommended changes in this week? Want to know for scheduling the Quick Start update?

Yes, I think so. We are working on it at the moment. It takes a while to test it due to number of templates and Kinesis/IoT options to test after changes.

**Comment by sshvans** [comment]
> > @EternalSH Do you think you will be able to have the recommended changes in this week? Want to know for scheduling the Quick Start update?
> 
> Yes, I think so. We are working on it at the moment. It takes a while to test it due to number of templates and Kinesis/IoT options to test after changes.

Sounds good. Thanks for the update!

File: README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
 ## AWS Industrial Time Series Data Connector on the AWS Cloud
 
 
-This Quick Start deploys the AWS Industrial Time Series Data Connector into your Amazon Web Services (AWS) account. The Industrial Time Series Data Connector synchronizes your on-premises systems to AWS to enable advanced analytics on your industrial process time series data. 
+This Quick Start deploys the AWS Industrial Time Series Data Connector into your Amazon Web Services (AWS) account. The connector synchronizes your OSIsoft PI System data to AWS to enable advanced analytics on your industrial process time-series data.
```
[the remark this answers is not in the document]
Comment (EternalSH) [reply README.md]: Yes, that was a leftover from our version of README from V2.

File: assets/api/run_app.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import logging
+
+from api.create_app import create_app
+
+app = create_app()
+
+if __name__ == "__main__":
+    app.logger.setLevel(logging.DEBUG)
+    app.run(host='0.0.0.0', port=4005, debug=True, threaded=True)
```
[the remark this answers is not in the document]
Comment (EternalSH) [reply assets/api/run_app.py]: This line is running only for local mode, used for testing (therefore also debug mode). It does not signify anything here.

[the remark this answers is not in the document]
Comment (sshvans) [reply assets/api/run_app.py]: Sounds good.

**Comment by sshvans** [comment]
@EternalSH The PR looks good and I have approved it. However, I noticed that the PR was made against master branch. I have changed it to be against the develop branch, however, I am seeing merge conflicts. Can you resolve those merge conflicts against develop branch and push the changes to the same PR, so that I can merge it?

**Comment by sshvans** [comment]
@EternalSH I actually realised that the conflict was on the submodule which can be ignored. So I went ahead and merged the PR.

**Pull request merged**